### PR TITLE
RF | FreestyleProjectRenameTest > TC_003.001

### PR DESCRIPTION
**Pull request by pojirakula**

### CODE REVIEW

**Comment by pojirakula** [comment]
[RF | FreestyleProjectRenameTest > TC_003.001](https://trello.com/c/qamPUqE1/565-rf-freestyleprojectrenametest-tc003001)

File: src/test/java/FreestyleProjectRenameTest.java [hunk 1]
```diff
@@ -13,13 +13,13 @@ public class FreestyleProjectRenameOlaTest extends BaseTest {
     private static final By DASHBOARD = By.xpath("//a[normalize-space(text())='Dashboard']");
     private static final By OK_BUTTON = By.xpath("//button[@id='ok-button']");
     private static final By SAVE_BUTTON = By.xpath(
-            "/html/body/div[5]/div/div/div/div/form/div[1]/div[12]/div/div[2]/div[2]/span[1]");
+            "//div[@class='bottom-sticker-inner']/span/span/button[@type='submit']");
```
Comment (RusAli) [inline src/test/java/FreestyleProjectRenameTest.java:16]: u can use By.id("yui-gen27-button")
or  By.xpath("//button[@id='yui-gen25-button']")

File: src/test/java/FreestyleProjectRenameTest.java [hunk 2]
```diff
@@ -13,13 +13,13 @@ public class FreestyleProjectRenameOlaTest extends BaseTest {
...
     private static final By FREESTYLE_PROJECT = By.xpath(
-            "//*[@id=\"j-add-item-type-standalone-projects\"]/ul/li[1]/div[1]");
+            "//div[@id='j-add-item-type-standalone-projects']/ul/li[1]/label/span");
     private static final By RENAME_FIELD = By.xpath(
-            "//*[@id=\"main-panel\"]/form/div[1]/div[1]/div[2]/input");
-    private static final By RENAME_BUTTON = By.xpath("//*[@id=\"yui-gen1\"]");
-    private String actualResult;
+            "//div[@id='main-panel']/form/div[1]/div[1]/div[2]/input");
+    private static final By RENAME_BUTTON = By.xpath(
+            "//div[@class='bottom-sticker-inner']/span/span/button");
```
Comment (RusAli) [inline src/test/java/FreestyleProjectRenameTest.java:22]: By.id("yui-gen1-button")
By.xpath("//button[@id='yui-gen1-button']")

**Review by RusAli — APPROVED** [review]

**Review by anonymoustuna — APPROVED** [review]

File: src/test/java/FreestyleProjectRenameTest.java (hunk 1 above)
Comment (pojirakula) [reply src/test/java/FreestyleProjectRenameTest.java:16]: Well, thanks

File: src/test/java/FreestyleProjectRenameTest.java (hunk 2 above)
Comment (pojirakula) [reply src/test/java/FreestyleProjectRenameTest.java:22]: Thank you!

**Pull request merged**